File: dot-net-projects-master/csasp-lms/UniversityLMS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace UniversityLMS.Controllers
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/

        public ActionResult Index()
        {
            try { TempData["UserName"] = Session["UserName"]; }
            catch (NullReferenceException ex) { /* No username set. */}
            return View();
        }
    }
}

File: dot-net-projects/vbasp-project-log/VBAspProjectLog/Controllers/HomeController.vb
<RAuthorizeAttribute()> _
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Private db As New ProjectLogContext

    '
    ' GET: /Home

    Function Index() As ActionResult

        'Get the projects data and feed into the view
        ViewBag.Projects = db.Projects.ToList()
        Return View()

    End Function

End Class